recipe/src/setup.py

```diff
@@ -5,7 +5,7 @@
     description = 'Read EDF files',
     py_modules = ['EdfFile'],
     name = 'edffile',
-    install_requires = ('python', 'numpy'),
+    install_requires = ('numpy'),
     license="MIT",
-    version = '5.0.0',
+    version = '5.0.1',
```

Comment:
I'm concerned about bumping the version number like this as it appears the entire package is 1 file and embedded in this feedstock.  Presumably where ever upstream is may have also had a 5.0.1 which is different?

That said, this looks like it was ripped out of pymca?  Maybe upstream does not think of this as a stand-alone project?

Reply:
Yes, there is a note about this in the recipe. I guess that means I shouldn't update the version number unless it is to the same version as the parent package which is now at 5.7.2.

This update is in a weird grey area because I made changes in the src folder, but the setup.py file does not exist in the upstream package. I guess I should be conservative and only update the build number.

Reply:
I have updated the recipe so the module source is now clearly linked to the version of the upstream package.